arcaflow_plugin_stressng/stressng_plugin.py

```diff
@@ -139,6 +159,48 @@ def to_jobfile(self) -> str:
         return result
 
 
+@dataclass
+class HDDStressorParams:
+    stressor: str
+    hdd: int = field(
+        metadata={
+            "name": "HDD workers",
+            "description": (
+                "start N workers continually writing, "
+                "reading and removing temporary files"
+            ),
+        }
+    )
+
+    hdd_bytes: str = field(
```

Comment:
In the future, we should switch this to a int with a unit.

Here is an example of using unit byte: https://github.com/arcalot/arcaflow-plugin-uperf/blob/bd89390237b72bf08fd1717c73300806a72ba9d5/arcaflow_plugin_uperf/uperf_schema.py#L155

The result is simply the code gets the base unit, so just plain bytes instead of kbytes, mbytes, etc.